packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -663,7 +664,8 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
               server.transformRequest(url, { ssr }).catch((e) => {
                 if (
                   e?.code === ERR_OUTDATED_OPTIMIZED_DEP ||
-                  e?.code === ERR_CLOSED_SERVER
+                  e?.code === ERR_CLOSED_SERVER ||
+                  e?.code === ERR_EISDIR
```

Comment:
I remember testing this PR a long time ago, but I'm not exactly sure if this is the right fix. I _think_ it's logged twice because once during, `ssrLoadModule`-ing the module, and once during `preTransformRequest`. And both can be transforming things at the same time?

It seems like a tricky situation to deduplicate things in this case, maybe it's better if the errors during pre-transforms here are prefixed with `Pre-transform error:`

Reply:
If I understand correctly, you mean to keep the error message output twice and add `Pre-transform` in front of the output message?

Reply:
Yeah. I think that's the safest way so we don't accidentally suppress errors during pre-transform